lib/dry/component/container.rb

```diff
@@ -123,6 +123,8 @@ def self.require(*paths)
       end
 
       def self.load_component(key)
+        return if key?(key)
```

Comment:
Oh sorry that wasn't a rhetorical question. I actually don't remember if we use this method in a way where return value is used :)